packages/react-components/src/components/VideoGallery/Layout.ts

```diff
@@ -30,7 +30,7 @@ export interface LayoutProps {
    * Maximum number of participant remote video streams that is rendered.
    * @defaultValue 4
    */
-  maxRemoteVideoStreams?: number;
+  maxRemoteVideoStreams: number;
```

Comment:
Still the same question - can we make this optional, if no one provide this - then we use 4 in the component internal logic?

Like 

const maxStreams = maxRemoteVideoStreams ?? 4;

Then use maxStreams in the component

Reply:
Yea, for internal component, this is less concern about it. When we are exporting this for external usage we might look back to api design and review it, should be good

Reply:
would there be a use case to export the layout? maybe like as a interface?